saas/backend/api/management/mixins.py

```diff
@@ -84,14 +84,24 @@ def verify_api_by_role(self, app_code: str, role_id: int, api: ManagementAPIEnum
         role_source = RoleSource.objects.filter(source_type=RoleSourceTypeEnum.API.value, role_id=role_id).first()
         # 角色来源不存在，说明页面创建或默认初始化的，非API创建，所以任何系统都无法操作
         if role_source is None:
-            raise exceptions.PermissionDenied(
-                detail=f"role[{role_id}] can not be operated by app_code[{app_code}], since role source not exists"
-            )
+            # 如果role是对应system_id的系统管理员，则其来源则默认为传入的系统
+            if (
+                system_id
+                and Role.objects.filter(id=role_id, type=RoleType.SYSTEM_MANAGER.value, code=system_id).exists()
+            ):
+                source_system_id = system_id
+
+            else:
+                raise exceptions.PermissionDenied(
+                    detail=f"role[{role_id}] can not be operated by app_code[{app_code}], since role source not exists"
```

Comment:
since role source not exists => role xxx is not created by something....

Reply:
DONE